client/src/view/tensegrity-control.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2019. Beautiful Code BV, Rotterdam, Netherlands
+ * Licensed under GNU GENERAL PUBLIC LICENSE Version 3.
+ */
+
+import * as React from "react"
+import { useEffect, useState } from "react"
+import { Nav, NavItem, NavLink, TabContent, TabPane } from "reactstrap"
+
+import { IFabricEngine } from "../fabric/fabric-engine"
+import { Physics } from "../fabric/physics"
+import { ISelection, Selectable } from "../fabric/tensegrity-brick-types"
+import { TensegrityFabric } from "../fabric/tensegrity-fabric"
+
+import { GlobalFabricPanel } from "./global-fabric-panel"
+import { PhysicsPanel } from "./physics-panel"
+import { TensegrityEditPanel } from "./tensegrity-edit-panel"
+
+export function TensegrityControl({engine, physics, fabric, constructFabric, selection, setSelection}: {
+    engine: IFabricEngine,
+    physics: Physics,
+    fabric?: TensegrityFabric,
+    constructFabric: (fabricCode: string) => void,
+    selection: ISelection,
+    setSelection: (s: ISelection) => void,
+}): JSX.Element {
+
+    const [activeTab, setActiveTab] = useState("1")
```

Comment:
Just make it a descriptive string instead of a number e.g. "physics"

Reply:
Ah yes. Actually I made it an enum, even better.